mifospay/src/main/res/layout/activity_mobile_verification.xml

```diff
@@ -89,18 +89,21 @@
                 app:ccp_autoDetectCountry="true"
                 app:ccp_autoFormatNumber="false"/>
 
-            <EditText
-                android:id="@+id/et_mobile_number"
-                android:layout_width="wrap_content"
-                android:layout_height="48dp"
-                android:layout_marginLeft="@dimen/value_5dp"
-                android:layout_weight="1"
-                android:background="@drawable/bg_et_round_border"
-                android:hint="@string/mobile_number"
-                android:inputType="number"
-                android:maxLength="@integer/telephone_numbers_max_length_standard"
-                android:paddingLeft="@dimen/value_15dp"
-                android:paddingRight="15dp"/>
+
+            <android.support.design.widget.TextInputLayout
+                style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
+                android:layout_width="160dp"
```

Comment:
Nice work but @haran2248 please use/create resource values from `dimens.xml`. For example, you can create a resource as _value_160dp_ inside `dimens.xml` and use it here as `@dimen/value_160dp`